ts/a11y/speech/WebWorker.ts

```diff
@@ -393,6 +393,7 @@ export class WorkerHandler<N, T, D> {
       node = adaptor.childNodes(node)[0] as N;
       adaptor.setAttribute(node, 'data-semantic-type', 'dummy');
       this.setSpecialAttributes(node, sid, '');
+      adaptor.setAttribute(node, 'data-speech-node', 'true');
```

Comment:
Although this works, I've handled it in a different location just to be sure it is always set when speech or Braille is available, rather than relying on the origin `treeitem` node, as currently done in semantic-enrich.